web/themes/joinup/prototype/sass/components/_form-table.sass

```diff
@@ -23,3 +23,14 @@ thead th.form-table__header
 .form-table__cell--icon-container
   width: 40px
   padding: 0
+
+#password-policy-status
+  @extend .form-wrapper
```

Comment:
Please try to find a way to use a mixin. [More details](https://csswizardry.com/2014/11/when-to-use-extend-when-to-use-a-mixin/).